app/controllers/claims/pages_controller.rb

```diff
@@ -1,2 +1,5 @@
 class Claims::PagesController < ApplicationController
+  def index
+    @school = User.first&.schools&.first&.gias_school
+  end
```

Comment:
If this isn't a meaningful lookup, let's use something more straight-forward with a comment.

```suggestion
  def index
    @school = Claims::School.first # Placeholder
  end
```